.changeset/beige-lamps-camp.md

```diff
@@ -0,0 +1,7 @@
+---
+'lit-html': minor
+'lit': minor
+'lit-element': minor
```

Comment:
Nothing here touches the `lit-element` package so I don't think it needs a bump there.
```suggestion
'lit-html': minor
'lit': minor
```

Reply:
I thought we were bumping `lit` and `lit-element` along with `lit-html` so that if you wanted to ensure you got a new feature but were only depending on `lit` or `lit-element` you could bump your own dependency version.

Reply:
Hmm.. I thought it was more so about making sure `lit` gets bumped, but yeah, I guess `lit-element` does have a dependency on `lit-html` so it would be a good signal for users that just depend on the `lit-element` package. Nevermind!